osu.Game/Online/Multiplayer/MultiplayerRoom.cs

```diff
@@ -39,14 +39,17 @@ public class MultiplayerRoom
         /// All users currently in this room.
         /// </summary>
         [Key(3)]
-        public List<MultiplayerRoomUser> Users { get; set; } = new List<MultiplayerRoomUser>();
+        public IList<MultiplayerRoomUser> Users { get; set; } = new List<MultiplayerRoomUser>();
 
         /// <summary>
         /// The host of this room, in control of changing room settings.
         /// </summary>
         [Key(4)]
         public MultiplayerRoomUser? Host { get; set; }
 
+        [Key(5)]
+        public MatchRoomState? MatchState { get; set; }
```

Comment:
Not sold on the nullability (here and the user's state), but I guess we can change this at any point.